repository/src/test/java/org/pentaho/platform/security/userroledao/jackrabbit/DefaultPentahoPasswordEncoderTest.java

```diff
@@ -0,0 +1,42 @@
+package org.pentaho.platform.security.userroledao.jackrabbit;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.pentaho.platform.repository2.userroledao.jackrabbit.security.DefaultPentahoPasswordEncoder;
+import org.pentaho.test.platform.security.userroledao.ws.UserRoleWebServiceBase;
+
+/*
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License, version 2 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ *
+ * Copyright 2006 - 2017 Pentaho Corporation.  All rights reserved.
```

Comment:
@pentaho-amessier @pentaho-esager I reached out to @mbatchelor for clarification, and 

- a) New files should have the "endYear" alone ( no need to have the "startYear" as well )
- b) New files that do happen to include the "old" pattern of "startYear - endYear" pattern **needn't** be changed/updated